pypeit/core/extract.py

```diff
@@ -626,19 +619,22 @@ def fit_profile(image, ivar, waveimg, trace_in, wave, flux, fluxivar,
 
      Parameters
      ----------
-     image : numpy float 2-d array [nspec, nspat]
+     image : numpy float 2-d array (nspec, nspat)
```

Comment:
these should all read something like:

image (np.ndarray): 2-d array (nspec, nsap)